src/sentry/api/endpoints/filechange.py

```diff
@@ -31,12 +32,17 @@ def get(self, request, organization, version):
         if not self.has_release_permission(request, organization, release):
             raise ResourceDoesNotExist
 
+        try:
+            limit = int(request.GET.get("limit"))
+        except ValueError:
+            limit = None
```

Comment:
Could we enforce a limit of sentry.api.paginator.MAX_LIMIT which is 100

Reply:
The way it worked in old releases for ages is that this endpoint returned ALL unpaginated results - now we want to show in the frontend first 200 and then there's a button called show all files which will load everything. I would like to revisit this behavior, this is more of a quick fix for one specific edge case.

Reply:
Sounds good.